scripts/cl/one_for_zero.py

```diff
@@ -2,30 +2,36 @@
 import sympy as sp
 from common import *
 
+
```

Comment:
note: all the python changes are from my prettifier. lmk if there's a specific preference and i'll reset them